rancher_pr_deploy.sh

```diff
@@ -110,28 +147,21 @@ deploy() {
     rancher rm --stop --type stack "${RANCHER_STACK_NAME}"
   fi
 
-  if rancher catalog install --answers answers.txt --name "${RANCHER_STACK_NAME}" "${RANCHER_CATALOG_NAME}"/"${RANCHER_TEMPLATE_NAME}"; then
-    DEPLOY_URL="http://${RANCHER_STACK_NAME}.dev.c2s.nhschoices.net"
-    MSG=":rocket: deployed to [${DEPLOY_URL}](${DEPLOY_URL})"
-    post_comment_to_github "$MSG" "$TRAVIS_PULL_REQUEST" "$TRAVIS_REPO_SLUG"
+  if rancher --wait catalog install --answers answers.txt --name "${RANCHER_STACK_NAME}" "${RANCHER_CATALOG_NAME}"/"${RANCHER_TEMPLATE_NAME}"; then
+    MSG="$(get_message "succeeded" "${RANCHER_STACK_NAME} in ${RANCHER_ENVIRONMENT}" "http://${RANCHER_STACK_NAME}.dev.c2s.nhschoices.net")"
+    eval "$COMMENTER \"$MSG\""
   else
-    MSG=":warning: deployment of ${TRAVIS_PULL_REQUEST} for ${TRAVIS_REPO_SLUG} to rancher stack ${RANCHER_STACK_NAME} failed"
-    post_comment_to_github "$MSG" "$TRAVIS_PULL_REQUEST" "$TRAVIS_REPO_SLUG"
+    MSG="$(get_message "failed" "${RANCHER_STACK_NAME} in ${RANCHER_ENVIRONMENT}")"
+    eval "$COMMENTER \"$MSG\""
     exit 1
   fi
 
   fold_end "Rancher_Up"
-}
 
+}
 
-REPO_NAME=$(get_repo_name "${TRAVIS_REPO_SLUG}")
-
-if [ "$TRAVIS" == true ] && [ "$TRAVIS_PULL_REQUEST" != false ] ; then
-  RANCHER_STACK_NAME="${REPO_NAME}-pr-${TRAVIS_PULL_REQUEST}"
-  IMAGE_TAG="pr-${TRAVIS_PULL_REQUEST}"
-  deploy
+if [ "$TRAVIS_PULL_REQUEST" != false ] ; then
```

Comment:
I've removed the test for TRAVIS as I don't think we need it and also as the beginning of "de-TRAVISification".

Let me know if you think we need it.